how-to-docs/kubernetes-robot-token.md

```diff
@@ -1,13 +1,26 @@
-# How to get a Kubernetes Robot Token for the UK cluster
+# How to get a Kubernetes Robot Token for a service in the UK cluster
 
-1. Log into the [platform hub][platform hub link].
+## Users
 
-2. Select **Support Requests** under **Help & Support** from the sidebar.
+1. Log into the [Platform Hub][platform hub link].
 
-3. Select **Kubernetes Robot Token Request** from the list of available requests.
+2. Go to the [Projects][project list] section and find your project. Click on the **Services** tab and find the service that requires a robot token.
 
-4. Enter the required information and submit your request.
+3. Go to the **Kube Robot Tokens** tab. Any robot tokens that have been created for that service will be listed. You can see the full token by clicking on the eye icon next to the token.
 
-5. Someone from the ACP platform team will process your request, after which you will find the robot token on the **Connected Identities** page on the platform hub.
+If there are no robot tokens for that service, or the required one is not there, you will need to ask your project admin(s) to create a robot token.
+
+## Project Admins (Creating a robot token)
+
+1. Log into the [Platform Hub][platform hub link].
+
+2. Go to the [Projects][project list] section and find your project. Click on the **Services** tab and find the service that requires a robot token.
+
+3. Go to the **Kube Robot Tokens** tab and click the **Create a Kubernetes robot token for this service** button.
+
+4. Select the required cluster, RBAC groups, robot name and description for the robot token and click **Create**.
```

Comment:
"What's my required RBAC group?"

Maybe we need to put some explanation around each of the fields here, and an example to refer to.

Reply:
Is there an example we could point to? or should we just make one up?

Reply:
I'd say just make one up.